tasks/test.py

```diff
@@ -675,7 +644,7 @@ def test(
         "covermode_opt": covermode_opt,
         "coverprofile": coverprofile,
         "test_run_arg": test_run_arg,
-        "timeout": timeout,
+        "timeout": int(timeout),
```

Comment:
:question: does this make a difference? I'd expect the result to be the same for a string and a number here

Reply:
I had the same intuition about this, but I think the goal is to raise an error in case someone inputs `10s` instead of `10`. [You can specify seconds (s) or minutes (m)](https://pkg.go.dev/cmd/go#hdr-Testing_flags) in the `-timeout` flag value, so I guess we're avoiding this edge case by int-ing it.

The case when this could cause problem is let's say someone inputs "10m" to have a 10 minutes timeout. Then the final command parsing `{timeout}s`, it would become `10ms` which is 10 milliseconds. That would be confusing aha